fury/ui/containers.py

```diff
@@ -776,7 +781,12 @@ def update_tabs(self):
 
             tab_panel.content_panel.resize(self.content_size)
             self.parent_panel.add_element(tab_panel, tab_panel_pos)
-            self.parent_panel.add_element(tab_panel.content_panel, (0.0, 0.0))
+            if self.tab_bar_pos.lower() == "top":
+                self.parent_panel.add_element(tab_panel.content_panel,
+                                              (0.0, 0.0))
+            elif self.tab_bar_pos.lower() == "bottom":
+                self.parent_panel.add_element(tab_panel.content_panel,
+                                              (0.0, 0.1))
```

Comment:
same question as above ?

Reply:
raising ValueError in the above part should take care of this also.

Reply:
ok, so do not forget to take care of this